guides/user/pipeline/managing-pipelines/index.md

```diff
@@ -0,0 +1,146 @@
+pipeline triggers and parameters. Each *stage* specifies an action the pipeline
+will take once it is configured.
+
+1. Select **Add stage** from your pipeline configuration screen.
+  ![](images/add-stage.png)
+2. Set the stage type using the drop-down menu.
+3. If this is not the first stage in your pipeline, make sure that this stage
+depends on the desired upstream stage(s) using the **Depends on** field.
+  ![](images/stage-depends-on.png)
+
+You can add as many stages as your pipeline needs, in any order that makes sense
+for you.
+
+### Add a trigger
+
+Make sure that you are editing the **Configuration** stage of your pipeline.
+
+![](images/configuration-stage.png)
+
+1. Select **Add trigger**.
+  ![](images/add-trigger.png)
+2. Choose your desired trigger type from the drop-down menu that appears, and
+input any further required configuration.
+
+For further information on how triggers work, see the documentation on pipeline
+triggers.
+<!-- TODO(nhayes): link here to pipeline triggers overview once it exists. -->
+
+## Disable a pipeline
+
+Disabling a pipeline prevents any triggers from firing, as well as preventing
+users from running it manually.
+
+1. From the **Pipelines** tab, click **Configure** to modify an existing
+pipeline.
+  ![](images/select-configure.png)
+2. Click **Pipeline actions** in the upper right corner, and select **Disable**.
+  ![](images/pipeline-actions.png)
+
+In order to re-enable your disabled pipeline, select **Pipeline actions** and
+choose **Enable**.
+
+## Delete a pipeline
+
+1. From the **Pipelines** tab, click **Configure** to modify an existing
+pipeline.
+  ![](images/select-configure.png)
+2. Click **Pipeline actions** in the upper right corner, and select **Delete**.
+  ![](images/pipeline-actions.png)
+
+## Edit a pipeline as JSON
+
+> **Warning**: the JSON you write here is not validated -- Edit as JSON
+essentially allows you to modify the pipeline via a free-form textbox. **Be
+careful!** It is very easy to break the pipeline.
+
+Spinnaker represents pipelines as JSON behind the scenes. Any changes you make
+to your pipeline using the UI are converted to JSON when Spinnaker saves the
+pipeline.
+
+When you use the **Edit as JSON** feature, you are directly editing the payload.
+This can be useful for working around limitations of the UI.
+
+In order to edit your pipeline as JSON:
+
+1. From the **Pipelines** tab, click **Configure** to modify an existing
+pipeline.
+  ![](images/select-configure.png)
+2. Click **Pipeline actions** in the upper right corner, and select
+**Edit as JSON**.
+  ![](images/pipeline-actions.png)
+
+## View and restore revision history
+
+Each time you save your pipeline, the current version is added to revision
+history. You can use revision history to diff two versions of a pipeline or to
+restore an older version of a pipeline.
+
+> *Note*: If you are using minio or redis to store your Spinnaker configuration
```

Comment:
How would a user or admin know if this is the case? Is this about having chosen minio or redis as your [storage service](https://www.spinnaker.io/setup/install/storage/)? If so, perhaps rephrase to use that wording?

Reply:
I'm actually not sure - @lwander, @jtk54 could you clarify?

Reply:
Yes it would have to be chosen explicitly during setup/configuration